### PR TITLE
Updated and renamed ADR for database to include Options Considered section

### PR DESCRIPTION
**Pull request by amymok**
## What does this change? 

- Renamed ADR for database to include the choice and use of active voice
- Added "Options Considered" section to resolve [comment](https://github.com/18F/OPRE-Unicorn/pull/58#pullrequestreview-701243277) from PR #58

## What does this change look like?

[Preview link for `004-use-postgresql-for-database.md`.](https://github.com/18F/OPRE-Unicorn/blob/aym/25/database/docs/adr/004-use-postgresql-for-database.md) 👁️ 

## GitHub cards

Completes part of #25.

### CODE REVIEW

File: docs/adr/004-use-postgresql-for-database.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ We will likely need a relational database since the data is mostly structured an
 
 Reference to [different types of databases and their usages](https://www.geeksforgeeks.org/types-of-databases/).
 
+## Options Considered
+
+We consider a few factors when we pick a database:
+- relational
```
Comment (carjug) [inline docs/adr/004-use-postgresql-for-database.md:21]: Nitpick: do we want to expound on a this a bit? Maybe stating that the data is relational, so we should go with a relational database? Just a thought!

**Review by carjug — APPROVED** [review]
LGTM! One small nitpick.

File: docs/adr/004-use-postgresql-for-database.md (hunk 1 above)
Comment (amymok) [reply docs/adr/004-use-postgresql-for-database.md]: Added!

**Review by ninamak — APPROVED** [review]
LGTM!

**Pull request merged**